kong/runloop/handler.lua

```diff
@@ -106,6 +107,7 @@ local set_upstream_ssl_trusted_store = ktls.set_upstream_ssl_trusted_store
 if is_http_module then
   set_authority = require("resty.kong.grpc").set_authority
   set_log_level = require("resty.kong.log").set_log_level
+  get_log_level = require("resty.kong.log").get_log_level
```

Comment:
Perhaps after this PR is merged, we could try to enable them in stream subsystem.

Reply:
sure